File: SDKs/zoom.us/2.0.0/python/src/sdk/models/operations/deleteaphonenumberslg.py
import dataclasses



@dataclasses.dataclass
class DeleteAPhoneNumberSlgPathParams:
    phone_number_id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'phoneNumberId', 'style': 'simple', 'explode': False }})
    shared_line_group_id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'sharedLineGroupId', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class DeleteAPhoneNumberSlgRequest:
    path_params: DeleteAPhoneNumberSlgPathParams = dataclasses.field()
    

@dataclasses.dataclass
class DeleteAPhoneNumberSlgResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    

File: SDKs/zoom.us/2.0.0/typescript/src/sdk/models/operations/deleteaphonenumberslg.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class DeleteAPhoneNumberSlgPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=phoneNumberId" })
  phoneNumberId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=sharedLineGroupId" })
  sharedLineGroupId: string;
}


export class DeleteAPhoneNumberSlgRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteAPhoneNumberSlgPathParams;
}


export class DeleteAPhoneNumberSlgResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
